frontend/src/lib/candidate/components/logoutButton/LogoutButton.svelte

```diff
@@ -7,8 +7,10 @@
   import {getContext} from 'svelte';
   import type {CandidateContext} from '$lib/utils/candidateStore';
 
-  /** Defaults to true, so that button variant is "icon". Can be set to false if the button should be of variant "main" */
-  export let variantIcon = true;
+  /** Defaults to normal, button variant can be "normal" for just text, "icon" for a button with an icon and "main" for a filled red box */
+  export let buttonVariant = 'normal';
+  /*Determines whether the user is taken to the login page after logging out or stays on the current page*/
+  export let stayOnPage = false;
 
   /** time until automatic logout for modal */
   export let logoutModalTimer = 30;
```

Comment:
As the component has become more complex I'd suggest that a `LogoutButton.type.ts` file is created with the type definitions and explanations, i.e.
```
export type LogoutButtonProps = {
  /**
   * Some text
   *
   * @default 'normal'
   */
  buttonVariant?: 'normal' | 'icon' | 'main';
  /**
   * ...
   *
   * @default false
   */
  stayOnPage?: boolean;
  /**
   * ...
   *
   * @default 30
   */
  logoutModalTimer?: number;
};
```
and then those are used instead in the component
```
import type {LogoutButtonProps} from './LogoutButton.type';

type $$Props = LogoutButtonProps;
export let buttonVariant: $$Props['buttonVariant'] = 'normal';
...
```
and so on